Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: woodpecker
 description: A Helm chart for Woodpecker CI
 type: application
-version: 0.1.5
-appVersion: "v0.15.9"
+version: 1.0.0
```

Comment:
As the chart itself does not have a breaking change etc, I would continue with `0.2.0`.